pkg/monitor/monitorapi/types.go

```diff
@@ -212,6 +216,8 @@ const (
 	UpgradeCompleteReason IntervalReason = "UpgradeComplete"
 
 	NodeInstallerReason IntervalReason = "NodeInstaller"
+
+	APIUnreachableIntervalReason IntervalReason = "APIUnreachableInterval"
```

Comment:
FromClient would be good to include here as well, and Reason should not mention "Interval". Our Reason's often seem to be equal to Sources when it will only contain one Reason. If anything, Source should be the more generic one if we plan for other "Reason's" to appear there someday.